src/java/com/blackNecoProject/pages/LoginPage.java

```diff
@@ -34,6 +35,11 @@ public String getLoginTitle() {
         return getText(titleText);
     }
 
+    public void alere()  {
```

Comment:
Use sleep for wait
Rename method for acceptAlertMessage
Adding try catch for noSuchAllertExeption